CMakeLists.txt

```diff
@@ -40,14 +43,5 @@ if(NOT CMAKE_SKIP_INSTALL_RULES)
   include(cmake/install-rules.cmake)
 endif()
 
-# ---- Developer mode ----
```

Comment:
Why are we unconditionally enabling dev mode?

Reply:
I thought you'd check for BUILD_X variables again anyway, so on first glance it seemed unnecessary. However I had a second look now and I see that you also add some lint and spell targets, so I'll add it back.